test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/TestEngineTests.cs

```diff
@@ -199,6 +199,29 @@ public void GetDiscoveryManagerShouldNotReturnsInProcessProxyDiscoveryManagereIf
         }
 
         [TestMethod]
+        public void GetDiscoveryManagerShouldNotReturnsInProcessProxyDiscoveryManagereIfrunsettingsHasTestSettingsInIt()
+        {
+            string settingXml =
+                @"<RunSettings>
+                    <RunConfiguration>
```

Comment:
Nit: not required Runconfiguration elements.

Reply:
Its required